### PR TITLE
Introduces new main functionality

### LINKED ISSUE #2: We need more text from main
Currently, main doesn't output very much info. We need more!

### PR DESCRIPTION
**Pull request by falquaddoomi**
- what does this PR do?
- what should reviewers review?
- what's missing or still needs to be fixed

Closes #2.

### CODE REVIEW

File: main.py
Comment (falquaddoomi) [inline main.py:1]: This file seems nice.

File: main.py [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 def main():
     print('Hello, World!')
+    print('Goodbye, world...')
+    print('Hello again, world...?')
```
Comment (d33bs) [inline main.py:6]: Nifty! 👍

Comment (falquaddoomi) [inline main.py:6]: These lines seem fishy...

File: main.py [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 def main():
     print('Hello, World!')
```
Comment (falquaddoomi) [inline main.py:4]: Let's not make this such a strong statement.

```suggestion
    print('Hello, World?')
```

File: main.py [hunk 3]
```diff
@@ -2,6 +2,8 @@
...
+    print('Goodbye, world...')
```
Comment (d33bs) [inline main.py:5]:
```suggestion
    print('Hello again, world...')
```

**Review by falquaddoomi — COMMENTED** [review]
This all seems great! Nice work!

**Review by d33bs — APPROVED** [review]
Great job! I left a few comments about improvements - feel free to reach out with questions.

**Pull request merged**